### PR TITLE
Restructuring of the reading system specification

### PR DESCRIPTION
**Pull request by mattgarrish**
This PR also sprung out of last week's telecon as I noticed that the spine discussion was split between elements only because that's the way the core spec was structured. There's no need to break topics like that along element lines.

From there, I noticed other quirks of pulling out the RS requirements that could stand for improvement:

I've removed all the "conformance" sections that were embedded within each subsection. I've either integrated these without the headings if they're the only thing in the section and contain actual requirements, or deleted them if all they contained were pointers to other sections (most of these lists were just helpful locators to where actual requirements were listed when the specs were merged so devs didn't have to read everything). Some of these statements were confusingly half-true (e.g., you "must" support scripting as defined in the scripting section when the scripting section doesn't actually require support for scripting).

Some of the other notable changes include:

- I've grouped three bullets in the main conformance section that all began with "if it has a viewport".
- I've noted that support for fixed layouts and media overlays is optional in the main conformance section (we used to say that support for fixed layout metadata is required, but this was never true as reading systems default to conformance, so I removed that statement)
- The support for "presentation logic" bullet in the package document conformance I've added as a subsection since I couldn't find any other place for it. It still needs to be returned to for #1516 
- The only bullet in the xhtml conformance section that wasn't redundant was support for ARIA. I've added this to the extensions section as a new section (HTML allows ARIA attributes but I can't find that it requires exposing them).
- I merged the requirements for handling author/user css changes with the conformance bullets and dropped the subsectioning.
- I've moved the informative section on scripting security to the security section at the end. These will likely need better integration in the future, though.

Also, the media overlays conformance section had one bullet that wasn't just a pointer to a subsection, but it was a bit confusing in that it says that xhtml content documents have to be supported and SVG may. I've made the obvious assumption this refers to playback, not support for the formats and worked this bullet into the loading section.

Long story short, the intention is only to simplify the spec by removing the unnecessary pointers and subsectioning. Actual requirements that have to be conformed to should all remain.

### CODE REVIEW

**Comment by mattgarrish** [comment]
- Reading Systems [preview](https://cdn.statically.io/gh/w3c/epub-specs/editorial/rs-reorg/epub33/rs/index.html)
- Reading Systems [diff](https://services.w3.org/htmldiff?doc1=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://w3c.github.io/epub-specs/epub33/rs/index.html&doc2=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://cdn.statically.io/gh/w3c/epub-specs/editorial/rs-reorg/epub33/rs/index.html)

File: epub33/rs/index.html [hunk 1]
```diff
@@ -280,280 +302,215 @@ <h2>EPUB Publications</h2>
...
-					</li>
-					<li>
-						<p id="confreq-rendition-rs-manifest">It SHOULD NOT use non-<a>Publication Resources</a> in the
-							rendering of an EPUB Publication due to the inherent limitations and risks involved (e.g.,
-							lack of information about the resource and how to process it, security risks from
-							remotely-hosted sources, lack of fallbacks, etc.).</p>
-					</li>
-				</ul>
+				<p id="confreq-rendition-rs-package">It MUST honor all presentation logic expressed through the <a
+						href="https://www.w3.org/TR/epub-33/#sec-package-def">Package Document</a> [[!EPUB-33]] (e.g.,
```
Comment (iherman) [inline epub33/rs/index.html]: This is a new section; maybe it reads better if "an EPUB Reading System" is used instead of "It"...

File: epub33/rs/index.html [hunk 2]
```diff
@@ -1230,20 +1050,32 @@ <h4>The <code>rendition:page-spread-*</code> Properties</h4>
...
+				<h3>Initial Containing Block Dimensions</h3>
 
-				<p>In this version of this specification, only the width and height expressions as defined in <a
-						href="https://www.w3.org/TR/epub-33/#sec-fxl-icb-html">Expressing in HTML</a> [[!EPUB-33]] MUST
-					be recognized by <a>Reading Systems</a>.</p>
+				<dl>
+					<dt>XHTML</dt>
+					<dd>
+						<p>In this version of this specification, <a>Reading Systems</a> MUST recognize only the width
+							and height expressions as defined in <a
```
Comment (iherman) [inline epub33/rs/index.html]: Why this "In this version of this specification"? I have not seen this type of reference elsewhere...

File: epub33/rs/index.html [hunk 3]
```diff
@@ -1930,6 +1683,89 @@ <h2>Security and Privacy</h2>
 				<p>The Working Group will have to address the particularity of cross-origin by addressing the question
 					of what the "origin" is for an EPUB Publication.</p>
 			</div>
+
+			<section id="sec-scripted-content-security">
+				<h4>Scripting Considerations</h4>
```
Comment (iherman) [inline epub33/rs/index.html:1684]: I wonder whether the editor's note preceding this section on the origin is still relevant. I believe that has been solved.

Comment (iherman) [reply epub33/rs/index.html:1684]: Actually, I wonder whether the editor's note referring to the pub manifest should still be kept, too. Maybe just to say that this section needs further work overall.

**Review by iherman — APPROVED** [review]
An additional comment: although, strictly speaking, this is not a substantive change, I think this editorial change should also be mentioned in the change log.

**Comment by mattgarrish** [comment]
> I think this editorial change should also be mentioned in the change log.

Ya, no harm. I was thinking being a new creation that structural reshuffling is par for the course, but now that we're past fpwd it is a notable change.

File: epub33/rs/index.html (hunk 2 above)
Comment (mattgarrish) [reply epub33/rs/index.html]: > Why this "In this version of this specification"?

I think that was added because there wasn't a firm definition of the viewport meta when we first introduced this. We were still citing the apple developer docs. Sort of a subtle warning that there might be additional requirements in the future.

I agree it's not really needed, as everything is subject to change in the future.

File: epub33/rs/index.html (hunk 1 above)
Comment (mattgarrish) [reply epub33/rs/index.html]: Oops, ya, missed that one.

File: epub33/rs/index.html (hunk 2 above)
Comment (mattgarrish) [reply epub33/rs/index.html]: I'm also going to tweak the wording so that it's the same as the SVG ICB text.

**Comment by mattgarrish** [comment]
I'm going to merge this so it doesn't become a blocker. I'm sure we'll continue to work at optimizing this document.
